cleanlab_studio/studio/studio.py

```diff
@@ -383,3 +386,30 @@ def poll_cleanset_status(self, cleanset_id: str, timeout: Optional[int] = None)
 
         except (TimeoutError, CleansetError):
             return False
+
+    def get_autofix_defaults(self, project_id: str) -> dict:
```

Comment:
this should be cleanset_id not project_id. 

All of the computations to get the values to use depend on the cleanset, not the original dataset/project

Reply:
Addressed here: 
https://github.com/cleanlab/cleanlab-studio/blob/d67bbc37c052929a2fbd134c57dcd62f2e24bb27/cleanlab_studio/studio/studio.py#L391